src/Microsoft.Azure.SignalR/Startup/AzureSignalRHostingStartup.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NETCOREAPP3_0
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.SignalR.Startup;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+[assembly: HostingStartup(typeof(AzureSignalRHostingStartup))]
+
+namespace Microsoft.Azure.SignalR.Startup
```

Comment:
Put this in the `Microsoft.Azure.SignalR` namespace and get rid of the .Startup.